src/createStore.ts

```diff
@@ -41,7 +41,13 @@ function createStore(history: History): Store<IState> {
           ...defaultState,
           session: {
             ...defaultState.session,
-            ...loadedStore
+            ...loadedStore.session
+          },
+          workspaces: {
+            ...defaultState.workspaces,
+            playground: loadedStore.playgroundWorkspace
```

Comment:
Do you think we should use the spread instead (`{...defaultPlaygroundWorkspace, ...loadedStore.playgroundWorkspace}`)? This will allow the stored playgroundWorkspace to be Partial as well. But perhaps it might be easier to work with the exclusive decision, especially when debugging

Reply:
Fixed with 2f0e5ca.